### PR TITLE
Adds GBP as a GitHub Action

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
https://github.com/tgstation/gbp-action

Old balances have been ported over as of Feb 24 2021, 10 PM PST. I will not be transferring over scores after that.

Keeps the one in the webhook in case this doesn't work. Yes, this means there will be 2 of the same system running commenting on people's PRs. When the action proves to work, I will remove the GBP code from the webhook. Eventually, the entire webhook will be eaten by actions, God willing.

## Why It's Good For The Game
Splits us from TGMC's balances (who should implement this themselves if they want to keep the system), gets us off a PHP webhook and completely in repo, supports changing names, etc.

### CODE REVIEW

**Comment by Mothblocks** [comment]
It's possible just due to the nature of GitHub actions that there will be some followup PRs after this to try and squash bugs, I really don't know.

It works fine on the gbp-action repo, so this is a wide scale test.

File: .github/gbp-balances.toml [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+27095287 = 4 # bigfatbananacyclops
+27370365 = 10 # imsxz
+27729769 = 4 # LorenzoTheMarauder
+27740099 = 5 # challade
+28007787 = -2 # sqnztb
+28513509 = 5 # ecronogist
+28731893 = 7 # Dionant
+28809925 = 22 # GLACoding
+28861083 = 4 # Ketrai
+29008542 = 43 # MrDoomBringer
```
Comment (MrDoomBringer) [inline .github/gbp-balances.toml:341]:
```suggestion
29008542 = 430 # MrDoomBringer
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]
Code looks good, just one minor syntax issue 👍

File: .github/gbp-balances.toml (hunk 1 above)
Comment (Mothblocks) [reply .github/gbp-balances.toml:341]: :robot: This is an automated message to let you know your commit has been accepted.

**Review by Cyberboss — APPROVED** [review]
In the interest of minimizing import drift, next maintainer merge this please.

**Comment by Cyberboss** [comment]
_insert cringe thanos gif_

**Comment by Qustinnus** [comment]
image
my lawyers will contact you soon.

**Pull request merged**